docs/general/manage-tokens/how-to-transfer-rose-erc20.md

```diff
@@ -0,0 +1,269 @@
+- Gas fees on source chain (ETH, BNB, etc.)
+
+:::tip
+Have some **ROSE on Sapphire** for gas fees to use your bridged tokens.
+See [Transferring ROSE](#transferring-rose-to-sapphire-from-exchanges-or-oasis-wallet) below.
+:::
+
+
+## Using Celer cBridge to Bridge Assets to Sapphire
+
+**Supported assets:** ETH, USDC, USDT, WBTC, BNB, MATIC, OCEAN, wROSE  
+**Supported networks:** Ethereum, Polygon, BNB Chain
+
+1.  **Open cBridge and Connect Wallet:** Go to the cBridge web app at
+    [**cbridge.celer.network**][cbridge]. Connect your wallet (MetaMask or
+    another Web3 wallet) to cBridge. Make sure your wallet is set to the
+    **source network** from which you want to bridge. For example, if bridging
+    from Ethereum Mainnet, switch to Ethereum in MetaMask; if from BNB Smart
+    Chain, switch to BNB Chain, etc.
+
+    ![Celer Bridge](../images/manage-tokens/celer-bridge.png)
+
+2.  **Select Source and Destination Chains:** In the cBridge interface, use the
+    drop-down menus to select your **“From” chain and “To” chain**. For example,
+    choose **Ethereum** as the source and **Oasis Sapphire** as the destination
+    to bridge from Ethereum to Sapphire.
+
+3.  **Choose the Asset to Bridge:** Next, select the token you want to
+    transfer. The available token list will update based on the selected
+    chains. For instance:
+
+    *   If bridging from Ethereum, you might choose **ETH** or a stablecoin
+        like **USDC**. Bridging ETH will result in **WETH on Sapphire**
+        (wrapped Ether on Oasis) , and bridging USDC will result in USDC on
+        Sapphire (address given by cBridge).
+
+    *   If bridging from BNB Chain to Sapphire, you can select **wROSE** (the
+        wrapped ROSE token on BNB Chain).
+
+4.  **Enter Amount and Transfer:** Enter the amount you wish to bridge. The
+    interface may display the estimated receive amount after fees. Click
+    **Transfer**. For ERC-20 assets, your wallet will first
+    ask you to **approve** the cBridge contract to spend that token (e.g.
+    approve USDC). Approve the token, then confirm the **bridging transaction**
+    in your wallet. This will send the tokens to cBridge on the source chain.
+
+5.  **Confirm and Wait:** After confirming, cBridge will handle the cross-chain
+    transfer. The bridging typically completes **within a few minutes**, but
+    times vary by chain. You can monitor the transfer status on the cBridge
+    interface.
+
+6.  **Receive Tokens on Sapphire:** Tokens arrive at your Sapphire address.
+    You may need to add the token contract address in MetaMask to see them.
+    See [Contract Addresses][token-addresses] for official token addresses.
+
+:::info
+cBridge is **non-custodial** - tokens are sent automatically after
+confirmation. Always use the **official URL** and verify **Oasis Sapphire** as
+destination.
+:::
+
+## Transferring ROSE to Sapphire (From Exchanges or Oasis Wallet)
```

Comment:
This is (or should) already be covered at https://deploy-preview-1328--oasisprotocol-docs.netlify.app/general/manage-tokens/#obtaining-rose. Maybe we should call the section there "Get ROSE" or maybe we could even devote a special chapter to it. In any case can you move this section there and just link it from here?